maintainers/maintainer-list.nix

```
@@ -13498,6 +13498,15 @@
     githubId = 619015;
     name = "Svintsov Dmitry";
   };
+  urandom = {
```

```suggestion
  arnottcr = {
```
Can you please use a name that is somehow related to your existing profile to make it easier to request review from or ping you?

---

my online identity is currently centered around the `urandom` handle, unfortunately my github account dates back to high school, and somebody else took that handle before I could correct things here.

you will also note my email domain is `urandom.co.uk`, and I am urandom on [discourse](https://discourse.nixos.org/u/urandom), [sourcehut](https://git.sr.ht/~urandom), and other platforms.